Source/NSView.m

```diff
@@ -5129,6 +5131,49 @@ - (void) setUserInterfaceLayoutDirection: (NSUserInterfaceLayoutDirection)dir
   return;
 }
 
+/**
+* Layout 
+*/
+
+- (void) layout
+{
+  // FIXME: Implement asking layout engine for the alignment rect of each subview and set the alignment rect of each sub view
+}
+
+- (void) layoutSubtreeIfNeeded
+{
+  [self updateConstraintsForSubtreeIfNeeded];
+  [self _layoutViewAndSubViews];
+}
+
+- (void) _layoutViewAndSubViews
+{
+  if (_needsLayout)
+    {
+      [self layout];
+      _needsLayout = NO;
+    }
+
+  NSArray *subviews = [self subviews];
+  FOR_IN (NSView *, subview, subviews)
+    [subview _layoutViewAndSubViews];
+  END_FOR_IN (subviews);
+}
+
+- (void) setNeedsLayout: (BOOL) needsLayout
+{
+  if (!needsLayout)
```

Comment:
You could compare with the current value here, but this is again wrong.

Reply:
This is another setter that can only be toggled on from what I observed. It can only be turned off by internal code.